docker-compose.yml

```diff
@@ -6,12 +6,39 @@ services:
       target: build
       args:
         nodeEnv: development
-    ports:
-      - "8686:8686"
-    depends_on:
-      - dgraph
     environment:
       DGRAPH_URL: http://dgraph:8080
       INTEGRATION_TEST: "true"
-  dgraph:
-    image: "dgraph/standalone:v21.03.2"
+    ports:
+      - "8686:8686"
+    depends_on:
+      - alpha
+  # Dgraph Zero controls the cluster
+  zero:
+    image: dgraph/dgraph:latest
+    container_name: lambda_dgraph_zero
+    volumes:
+      - ~/local-dgraph-data:/dgraph
+    ports:
+      - 5080:5080
+      - 6080:6080
+    command: dgraph zero --my=zero:5080 --logtostderr -v=2 --telemetry sentry=false
+    restart: unless-stopped
+  # Dgraph Alpha hosts the graph and indexes
+  alpha:
+    image: dgraph/dgraph:latest
+    container_name: lambda_dgraph_alpha
+    volumes:
+      - ~/local-dgraph-data:/dgraph
+    ports:
+      - 8080:8080
+      - 9080:9080
+    command: >
+      dgraph alpha --my=alpha:7080 --zero=zero:5080
+        --security whitelist=0.0.0.0/0
+        --logtostderr -v=2
+        --graphql lambda-url=http://host.docker.internal:8686/graphql-worker
+        --telemetry sentry=false
+    restart: unless-stopped
```

Comment:
depends on zero?

Reply:
Copy from Just used the https://github.com/dgraph-io/dgraph/blob/main/contrib/local-test/docker-compose.yml
If it needs a dependency on zero, please add it.